src/templates/prototypes/_toggles.pug

```diff
@@ -44,3 +44,29 @@ block content
           dt Website:
           dd #[+link('#', 'www.conservatives.com')]
         p You may be able to discuss issues with your MP in person or online. Contact them for details.
+
+      h2 Radio Switch (content initally shown)
+      
+      +form('javascript:console.log("form submitted")', 'post')
```

Comment:
random console.log?

Reply:
I followed on from the ways other dummy forms have been implemented - see form elements and search results pages. Happy to change to another approach if required.